wp-content/themes/core/components/image/css/image.pcss

```diff
@@ -22,14 +13,15 @@
 .c-image__link {}
 
 /* -----------------------------------------------------------------------------
- * Image Component: Image Element
+ * Image Component: Wrapper element
  *
- * An `img` for foreground usages and a `div` for background usages.
+ * Generally, a `figure` for foreground usages and a `div` for background usages.
+ * Customizable via component's attributes.
  * ----------------------------------------------------------------------------- */
 
-.c-image__image {
+.c-image {
 	/* CASE: Background image implementation. */
-	.c-image--bg & {
+	&.c-image--bg .c-image__image {
```

Comment:
To un-nest the selectors here, this can use a parent-modifer such as:

.c-image__image { 

	/* CASE: Background image implementation */
	.c-image--bg & {